src/main/scala/dpla/ingestion3/model/DplaMapData.scala

```diff
@@ -131,5 +131,6 @@ case class EdmTimeSpan(
 
 case class URI(value: String) {
   def validate: Boolean = Try { new java.net.URI(value) }.isSuccess
-  def print: String = value
+
+  override def toString: String = value
```

Comment:
Could you add a comment here explaining why you needed to do this?  I can imagine in six months I won't remember.

Reply:
Done.